ports/cpp/source/antlr4-c3/CodeCompletionCore.hpp

```diff
@@ -110,44 +129,49 @@ class CodeCompletionCore {
    * Tailoring of the result:
    * Tokens which should not appear in the candidates set.
    */
-  std::unordered_set<size_t> ignoredTokens;  // NOLINT: public field
+  void setIgnoredTokens(std::unordered_set<size_t>&& ignoredTokens);
```

Comment:
I see no reason why a public field could be harmful in any way, if it has no side effect when setting it. Additionally, it's a result field, so it is supposed to be a simple public field!